src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css

```diff
@@ -232,7 +232,7 @@ html[data-theme="light"] .bd-sidebar-primary .theme-switch-button span {
   --bs-dropdown-border-radius: 0rem;
   background-color: var(--sst-header-background);
   box-shadow: none;
-  margin-top: 0.85rem;
+  margin-top: 0.86rem;
```

Comment:
haha 0.01 rem,

Reply:
it was like 1px out at normal zoom for me lol